app/forms/YesNoPageFormProvider.scala

```diff
@@ -25,7 +25,7 @@ import javax.inject.Inject
 
 class YesNoPageFormProvider @Inject()() {
 
-  protected[forms] val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,.@/]+$"""
+  protected[forms] val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,./]+$"""
```

Comment:
Hi, it seems need '@' sign in the regex, I am not sure why this has been removed. In the updated error message also design team expecting the 'atʼ signs (@)'

Reply:
@ symbol should be accepted , it should be there as part of regex